src/test/snapshot/test_snapshot_graphql.js

```diff
@@ -0,0 +1,246 @@
+    "0xd647b65fF1e4b25E3d2Dc86553A42c2675072A1a":16500,
+    "0x5cFDBd9150596b790AAe28e003EFfac9f29081F9":5000,
+    "0x12BD31628075C20919BA838b89F414241b8c4869":23804.9,
+    "0xaaC3438D37Da2FA2cE9A91c43834757314A2211b":1042.4945485487,
+    "0xF3E6e119972D9FBC06D34919a5EeCE4Ab5E47975":100,
+    "0x0B4a3351096b93672EFe48058A2De00E520F50f9":1471.0523646851,
+    "0x206120ebC24119b650B58df6Fc8559C8418A9e83":488.176503231012,
+    "0xF0e5094e51CDD81eBD68dD3F19AA580E2396A94D":10792.9526767501,
+    "0x85799fF1c86f89FE9fB07773d1d240763b6039Ce":201.612019392581,
+    "0x1cA48a32482eF29708e95F248E8a613f05782ED3":16149.2983099485,
+    "0x9b315a6B563687357FaF7493700C3EA406f5d55c":139.249169580784,
+    "0x2D3fb3096F020029A49A45c88253A4CD0BC2503F":1532.23026721296,
+    "0xB3D31eCf4C702Dc516c7a7e67F385560eE04A9C3":15018.9080305644,
+}
+
+const granularVoting_blockHeight = 12968040;
+const granularVoting_proposalIPFSHash = "QmWQQqJbdLYTW5sU3Khif1gxmeEXoujRzwzHP5uJiPwX9h"
+const granularVoting_voterValidation = {
+    "0x48b576c87e788a762D6f95A456f2A39113b46950":20.2657445,
+    "0x227FD2fD881Cc6c99DFCcc0FB40f2B1dc2f3F36E":3793.338596,
+    "0xbbd33AFa85539fa65cc08A2e61a001876D2f13FE":42957.92109,
+    "0x8Fcf1CA67248A96e63A63Ea125e1fD91D47da4f5":1250,
+    "0x0bc9CD548cc04Bfcf8ef2Fca50c13b9b4F62f6D4":1250,
+    "0xBF6DEDC1bB233e7E8E8f085227eAA1d611103A68":1250,
+    "0x008ed443F31a4b3aEe02fbFe61c7572dDaf3A679":1100.2054,
+    "0x2BcA3836741FE09f690ca7F1EDfCED65968D57C3":33532.8,
+    "0x61Ae4E8691335569776eEAd72D16f4f002E64a56":1979.856468,
+    "0x7a791fDA775B399f4A78f68a0b66498e6bcD3492":0.0999,
+    "0x87E4B99cE07073594AdDA44338705BA1Ab674cbD":34.50403844,
+    "0x105DfEd176B4c5Ba488Ee5F4362638Ab0216b8c4":90681.41938,
+    "0xF4B41aD3165d64AAb755F0f325bAF67F84bE28E5":270.8918581,
+    "0x01e66950353400E93AEe7F041C0303103E2ef5Ab":22695.49488,
+    "0x362CfE20851584DF00a670b2c8460A3aafD35839":30000,
+    "0xd71B984f8028af956fD5dAcE23Dd4f90475a82CC":49999.73335,
+    "0xF49A4ba03f69eEf2366303e64Fc7b3a33d3f0043":25503.25545,
+    "0x1a0c2621E983CA01c2ccd1e2898E65e5a0775c54":19986.05027,
+    "0xB39EC688A27B14436e9F15A693Da8605A3D0269D":350000.0002,
+    "0xCbD14C1a2593Eeeef38AC8d3d9E3df6aa405b4eC":100000,
+    "0xCbD2c001A4450b4dAD7920951bcFd7a328fEAEa6":450000,
+    "0x67B69C822553FE57d8B7E354af4f540c4e8963c9":26365.77007,
+    "0xE0dc24A3d7478eB840dC63baa20fcB06cdB123BE":386496.3534
+}
+
+const uni_sushi_bancor_blockNumber = 13060882;
+const uni_sushi_bancor_proposalIPFSHash = "QmQgfxvLqz88pL3ByK6U82bxezCU8MAiCSgxtTTCoR3fWm"
+const uni_sushi_bancor_voterValidation = {
+    "0x48b576c87e788a762D6f95A456f2A39113b46950":20.2334355075387,
+    "0x002570980aA53893C6981765698b6eBAB8aE7EA1":24318.582,
+    "0x655eFe6Eb2021b8CEfE22794d90293aeC37bb325":300329.407284156,
+    "0x61B15998893cC746B46C08FEdEE13a0d1b33bBa9":73013.1442325089, // This number is not matching to voting results => 73013.1442325089
```

Comment:
This is frying my brain. 
The test is failing because the score calculation does not match to what happened in snapshot.

You can see the voting scores based on the ETH blockHeight below.
https://snapshot.org/#/officialoceandao.eth/proposal/QmQgfxvLqz88pL3ByK6U82bxezCU8MAiCSgxtTTCoR3fWm